Assets/MRTK/SDK/Profiles/DefaultMixedRealityControllerVisualizationProfile.asset

```diff
@@ -73,3 +73,25 @@ MonoBehaviour:
     controllerVisualizationType:
       reference: Microsoft.MixedReality.Toolkit.Input.WindowsMixedRealityControllerVisualizer,
         Microsoft.MixedReality.Toolkit.SDK
+  - description: Windows Mixed Reality Articulated Hand
+    controllerType:
+      reference: Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input.WindowsMixedRealityArticulatedHand,
+        Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality
+    handedness: 1
+    useDefaultModel: 0
+    defaultModelMaterial: {fileID: 0}
+    overrideModel: {fileID: 1227372834072826, guid: 378f91dbbff9e2343b27a467467750bf,
+      type: 3}
+    controllerVisualizationType:
+      reference: Microsoft.MixedReality.Toolkit.Input.RiggedHandVisualizer, Microsoft.MixedReality.Toolkit.SDK
+  - description: Windows Mixed Reality Articulated Hand
+    controllerType:
+      reference: Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input.WindowsMixedRealityArticulatedHand,
+        Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality
+    handedness: 2
+    useDefaultModel: 0
+    defaultModelMaterial: {fileID: 0}
+    overrideModel: {fileID: 1788309537287834, guid: 132c9402d5843aa4f94380840186fd41,
+      type: 3}
+    controllerVisualizationType:
+      reference: Microsoft.MixedReality.Toolkit.Input.RiggedHandVisualizer, Microsoft.MixedReality.Toolkit.SDK
```

Comment:
ohhhhh yeah assigning the type of the visualizer might be what's causing the tests to mistakenly looking for a hand renderer reference

Reply:
Or, looks like this should be BaseHandVisualizer, to match what's already on the skeleton prefab?

Reply:
Looking through the code, this setting is ignored in most cases, including when there's an override prefab. So, I think this can be `None` with the skeleton prefab set

Reply:
Going to leave it as BaseHandVisualizer, in case we revisit this setting later and end up wanting to make functional changes (coming soon with the profile system work hopefully!)